src/obstacle_avoidance/include/MyNode.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Created By: Gareth Ellis
+ * Created On: July 16th, 2016
+ * Description: An example node that subscribes to a topic publishing strings,
+ *              and re-publishes everything it receives to another topic with
+ *              a "!" at the end
+ */
+
+#ifndef SAMPLE_PACKAGE_MYNODE_H
+#define SAMPLE_PACKAGE_MYNODE_H
+
+#include "geometry_msgs/Twist.h"
+#include "sensor_msgs/LaserScan.h"
+#include <cmath>
+#include <iostream>
+#include <ros/ros.h>
+#include <sb_utils.h>
+#include <std_msgs/String.h>
+
+#define LIDAR_ARRAY_SIZE 100
```

Comment:
I don't think we need this, the Lidar "array" is a Vector, so you can get the size with something like `myvec->size()`